app/src/test/java/gov/va/vro/VroControllerTest.java

```diff
@@ -124,34 +123,7 @@ void postFullHealthAssessment() throws Exception {
   }
 
   @Test
-  @DirtiesContext
-  void fullClaimSubmit_missing_evidence() throws Exception {
-    adviceWith(
-        camelContext,
-        "claim-submit",
-        route ->
-            route
-                .interceptSendToEndpoint(
-                    "rabbitmq:claim-submit-exchange"
-                        + "?queue=claim-submit"
-                        + "&routingKey=code.7101&requestTimeout=60000")
-                .skipSendToOriginalEndpoint()
-                .to("mock:claim-submit"));
-    // Mock secondary process endpoint
-    adviceWith(
-        camelContext,
-        "claim-submit-full",
-        route ->
-            route
-                .interceptSendToEndpoint(
-                    "rabbitmq:health-assess-exchange"
-                        + "?routingKey=health-assess.7101&requestTimeout=60000")
-                .skipSendToOriginalEndpoint()
-                .to("mock:claim-submit-full"));
-
-    mockFullHealthEndpoint.whenAnyExchangeReceived(
-        FunctionProcessor.<Claim, String>fromFunction(claim -> util.claimToResponse(claim, false)));
-
+  void fullHealthAssessmentMissingEvidence() throws Exception {
```

Comment:
It looks like this test failed.
Gradle Test Run :app:test > Gradle Test Executor 3 > VroControllerTest > fullHealthAssessmentMissingEvidence() FAILED
[150](https://github.com/department-of-veterans-affairs/abd-vro/actions/runs/3334337735/jobs/5517189235#step:5:151)
    org.opentest4j.AssertionFailedError at VroControllerTest.java:134

Reply:
@wlin2019 I cannot reproduce this locally and I do not think build check caught it either. Can you pull and try again?

Reply:
@wlin2019 I put everything back. not sure why that fixes it but it does. so please take another look.

Reply:
It works now.